src/Session.cpp

```diff
@@ -2,7 +2,10 @@
 #include <fstream>
 
 Session::Session(const std::string &configFilePath)
-: content(std::vector<Watchable*>()), actionsLog(std::vector<BaseAction*>()), userMap(std::unordered_map<std::string, User*>()), activeUser(nullptr)
+: content(std::vector<Watchable*>()),
+ actionsLog(std::vector<BaseAction*>()),
+ userMap(std::unordered_map<std::string, User*>()),
+ activeUser(nullptr)
```

Comment:
move the ":" next to `Session::Session(const std::string &configFilePath)`
+ initialization list should be indented